### PR TITLE
Add sysupdate verb

### PR DESCRIPTION
**Pull request by DaanDeMeyer**

### CODE REVIEW

File: mkosi/sysupdate.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    run(
+        cmd,
+        stdin=sys.stdin,
+        stdout=sys.stdout,
+        env=os.environ | config.environment,
+        log=False,
+        sandbox=config.sandbox(
+            binary=sysupdate,
+            devices=True,
+            network=True,
```
Comment (behrmann) [inline mkosi/sysupdate.py:43]: Do we need to allow network here? I'd gather that in the mkosi case everything will be local?

**Review by behrmann — COMMENTED** [review]
This is still missing a man page entry.

File: mkosi/sysupdate.py (hunk 1 above)
Comment (DaanDeMeyer) [reply mkosi/sysupdate.py:43]: Not necessarily, you have to do `PathRelativeTo=explicit` to look things up relative to the mkosi output directory so I'd prefer also allowing definitions that look up stuff remotely if that's what users want.

File: mkosi/resources/man/mkosi.md [hunk 2]
```diff
@@ -1699,6 +1708,21 @@ boolean argument: either `1`, `yes`, or `true` to enable, or `0`, `no`,
...
+    exists in the local directory, it will be used for this purpose as
+    well.
+
+    Note that `mkosi sysupdate` invokes `systemd-sysupdate` with
+    `--transfer-source=` set to the mkosi output directory. To make use
+    of this in a transfer definition file, set `PathRelativeTo=explicit`
+    to have the `Path=` setting for the transfer source be interpreted
+    relative to the mkosi output directory. Generally, configuring
+    `PathRelativeTo=explicit` and `Path=/` for the transfer source is
+    sufficient for the match pattern to be looked up relative to the
```
Comment (keszybz) [inline mkosi/resources/man/mkosi.md:1723]: s/looked up/interpreted/ ?

File: mkosi/sysupdate.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+from pathlib import Path
+
+from mkosi.config import Args, Config, OutputFormat
+from mkosi.log import die
+from mkosi.run import run
+from mkosi.types import PathString
+
+
+def run_sysupdate(args: Args, config: Config) -> None:
+    if config.output_format != OutputFormat.disk:
```
Comment (keszybz) [inline mkosi/sysupdate.py:14]: What about OutputFormat.directory? Wouldn't that work?

**Review by keszybz — APPROVED** [review]
LGTM.

**Pull request merged**